packages/block-editor/src/components/inspector-advanced-controls/index.js

```diff
@@ -13,7 +16,11 @@ const { Fill, Slot } = createSlotFill( name );
 
 function InspectorAdvancedControls( { children } ) {
 	const { isSelected } = useBlockEditContext();
-	return isSelected ? <Fill>{ children }</Fill> : null;
+	return isSelected ? (
+		<StyleProvider iframeDocument={ document }>
```

Comment:
What if we changed `iframeDocument` to just be `document`? At this point it's not just for iframes 😞

Reply:
yes, it makes sense :)